pyproject.toml

```diff
@@ -4,7 +4,7 @@ build-backend = "setuptools.build_meta"
 
 [project]
 name = "ops.manifest"
-version = "1.5.0"
+version = "1.6.0"
```

Comment:
Are we bumping the major version to account the breaking changes in `list_resources`?

Reply:
Ben had the same question.   I'm not sure that returning something when previously we weren't returning is really a `broken contract` on the API.  

I'm strongly confident there are no uses of `list_resources` where it checks for a `None` return value in the return value.  

But -- now that you and Ben bring it up, we can go to `2.0.0`.  I'd rather burn that `2.x` for some much better features though

Reply:
I removed the breaking change API